src/mito2/src/worker/handle_alter.rs

```diff
@@ -167,10 +178,89 @@ impl<S> RegionWorkerLoop<S> {
+                            let runs = parse_usize_with_default(
+                                &key,
+                                &value,
+                                TwcsOptions::default().max_active_window_runs,
+                            )?;
+                            options.max_active_window_runs = runs;
+                        }
+                        mito_engine_options::TWCS_MAX_ACTIVE_WINDOW_FILES => {
+                            let files = parse_usize_with_default(
+                                &key,
+                                &value,
+                                TwcsOptions::default().max_active_window_files,
+                            )?;
+                            options.max_active_window_files = files;
+                        }
+                        mito_engine_options::TWCS_MAX_INACTIVE_WINDOW_RUNS => {
+                            let runs = parse_usize_with_default(
+                                &key,
+                                &value,
+                                TwcsOptions::default().max_inactive_window_runs,
+                            )?;
+                            options.max_inactive_window_runs = runs;
+                        }
+                        mito_engine_options::TWCS_MAX_INACTIVE_WINDOW_FILES => {
+                            let files = parse_usize_with_default(
+                                &key,
+                                &value,
+                                TwcsOptions::default().max_inactive_window_files,
+                            )?;
+                            options.max_inactive_window_files = files;
+                        }
+                        mito_engine_options::TWCS_MAX_OUTPUT_FILE_SIZE => {
+                            let size = if value.is_empty() {
+                                TwcsOptions::default().max_output_file_size
+                            } else {
+                                Some(ReadableSize::from_str(&value).map_err(|_| {
+                                    InvalidRegionOptionChangeRequestSnafu { key, value }.build()
+                                })?)
+                            };
+                            options.max_output_file_size = size;
+                        }
+                        mito_engine_options::TWCS_TIME_WINDOW => {
+                            let window = if value.is_empty() {
+                                TwcsOptions::default().time_window
+                            } else {
+                                Some(humantime::parse_duration(&value).map_err(|_| {
+                                    InvalidRegionOptionChangeRequestSnafu { key, value }.build()
+                                })?)
+                            };
+                            options.time_window = window;
+                        }
+                        mito_engine_options::TWCS_FALLBACK_TO_LOCAL => {
```

Comment:
`TWCS_FALLBACK_TO_LOCAL` is also linked to remote compaction. If we don't allow changing the remote compaction option on the fly, then `TWCS_FALLBACK_TO_LOCAL` should not be allowed either.